### PR TITLE
Backoffice - typography - stylesheet

### PR DESCRIPTION
**Pull request by mykas**

### CODE REVIEW

File: packages/wix-ui-core/src/themes/backoffice/button/button-temp.st.css [hunk 1]
```diff
@@ -25,14 +25,21 @@
...
+
+BackofficeTheme .button {
   box-sizing: border-box;
   text-align: center;
   border: 1px solid;
 
-  font-family: value(fontRoman);
-  font-size: 16px;
-  line-height: 1.5;
+  -st-mixin: textMedium, textNormal;
```
Comment (argshook) [inline packages/wix-ui-core/src/themes/backoffice/button/button-temp.st.css]: @mykas  this line looks strange, it's both medium and normal.

Comment (mykas) [reply packages/wix-ui-core/src/themes/backoffice/button/button-temp.st.css]: tru tru

**Comment by hepiyellow** [comment]
Sorry, i meant to approve only the last commit.

**Comment by hepiyellow** [comment]
@mykas 
I didn't got over all the code. only the last commit.

**Pull request merged**